src/forms/CurrentExpenses.js

```diff
@@ -41,203 +41,323 @@ import { getUnder13Expenses } from '../utils/cashflow';
+      <CashFlowInputsRow
+        { ...sharedProps }
+        generic={ 'childTransportation' }>
+        { snippets.unreimbursedNonMedicalChildCare.childTransportation.i_label }
+      </CashFlowInputsRow>
+      <CashFlowInputsRow
+        { ...sharedProps }
+        generic={ 'childOtherCare' }>
+        { snippets.unreimbursedNonMedicalChildCare.childOtherCare.i_label }
+      </CashFlowInputsRow>
+
+      <EarnedFrom
+        hasExpenses = { getUnder13Expenses(current) !== 0 }
+        label    = { `If you didn't have that child care, would it change how much pay you can bring home?` }
+        propData = {{
+          client:        current,
+          childPropName: `earnedBecauseOfChildCare`,
+          update:        updateClientValue,
+        }}
+        CashFlowRow = {
+          <CashFlowInputsRow
+            { ...sharedProps }
+            generic = { `earnedBecauseOfChildCare` }>
+            { `How much less would you make?` }
+          </CashFlowInputsRow>
+        } />
 
     </div>
+  );
+};  // Ends <Under13>
 
+
+const ChildSupport = function ({ type, sharedProps }) {
+  return (
+    <div>
+      <ContentH1>Child Support</ContentH1>
+      <IntervalColumnHeadings type={ type } />
+      <CashFlowInputsRow
+        { ...sharedProps }
+        generic={ 'childSupportPaidOut' }> <strong>Legally obligated</strong> child support
+      </CashFlowInputsRow>
+    </div>
   );
-};  // End Utilities(<>)
+};
 
 
-const HousingDetails = function ({ current, type, time, updateClientValue }) {
+const DependentsOver12 = function ({ type, sharedProps }) {
+  return (
+    <div>
+      <ContentH1 subheading = { 'For the care of people who are older than 12, but are still dependents (those under 18 or disabled). Don\'t include amounts that are paid for by other benefit programs.\n' }>
+        Dependent Care of Persons Over 12 Years of Age
+      </ContentH1>
+      <IntervalColumnHeadings type={ type } />
+      <CashFlowInputsRow
+        { ...sharedProps }
+        generic={ 'adultDirectCare' }> Direct care costs
+      </CashFlowInputsRow>
+      <CashFlowInputsRow
+        { ...sharedProps }
+        generic={ 'adultTransportation' }> Transportation costs
+      </CashFlowInputsRow>
+      <CashFlowInputsRow
+        { ...sharedProps }
+        generic={ 'adultOtherCare' }> Other care
+      </CashFlowInputsRow>
+    </div>
+  );
+};  // Ends <DependentsOver12>
 
-  let housing = current.housing,
-      sharedProps = {
-        timeState:         current,
-        current:           current,
-        type:              type,
-        time:              time,
-        updateClientValue: updateClientValue,
-      };
 
-  if (current.housing === 'voucher') {
-    return (
-      <div>
-        <ContractRentField { ...sharedProps } />
-        <RentShareField { ...sharedProps } />
-        <Utilities { ...sharedProps } />
-      </div>
-    );
+const ElderlyOrDisabled1 = function ({ current, type, sharedProps, updateClientValue }) {
```

Comment:
Can we give this a more descriptive name than suffixing it with "1"? Maybe something like `ElderlyOrDisabledAssistance` and the component below as `UnreimbursedMedicalExpenses`?